Makefile

```diff
@@ -99,6 +99,7 @@ USE_MODULE_DB=true
 REGRESS = $(patsubst test/sql/%.sql,%,$(TESTS))
 REGRESS_OPTS = \
 	$(TEST_INSTANCE_OPTS) \
+	--create-role=super_user,regular_user,regular_user2 \
 	--inputdir=$(TEST_DIR) \
```

Comment:
Should set `--dbname=$TEST_DBNAME` for consistency. Then add `TEST_DBNAME`  to the exported values above so that they are visible to the `runner.sh` script.